src/test/java/DataAndPropertyProvider.java

```diff
@@ -22,41 +22,36 @@
         int rowNum = row.length;
         int colNum = row[0].split(",").length;
 
-        Object[][] data = new Integer[rowNum][colNum];
+        Object[][] data = new String[rowNum][colNum];
 
         for (int i = 0; i < rowNum; i++) {
             String[] cols = row[i].split(",");
 
             for (int j = 0; j < colNum; j++) {
-                String string = cols[j].trim();
-                data[i][j] = Integer.valueOf(string);
-                System.out.println("value is " + data[i][j]);
+                String string = cols[j];
+                data[i][j] = String.valueOf(string);
+               // System.out.println("value is " + data[i][j]);
             }
         }
         return data;
     }
 
-    private static Object[][] getDataSets(String filePath){
-        Object[][] returnObjArray = null;
+    private static Object[][] getDataSets(String filePath) {
+        //Object[][] returnObjArray=null;
         File file = new File(filePath);
-        try {
-            returnObjArray = readFile(file);
-        } catch (IOException e) {
-            e.printStackTrace();
-        }
+        Object[][] returnObjArray = readFile(file);
         return returnObjArray;
```

Comment:
Optional: just "return readFile(file);", there is no needs to create variable and then return it.